.uberenv_config.json

```diff
@@ -9,7 +9,7 @@
 "spack_packages_path": "scripts/spack/packages",
 "spack_concretizer": "clingo",
 "vcpkg_url": "https://github.com/microsoft/vcpkg",
-"vcpkg_commit": "14e7bb4ae24616ec54ff6b2f6ef4e8659434ea44",
+"vcpkg_commit": "cef0b3ec767df6e83806899fe9525f6cf8d7bc91",
```

Comment:
This corresponds to the `2022.06.16.1` release of vcpkg -- 
https://github.com/microsoft/vcpkg/releases/tag/2022.06.16.1